docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -353,4 +353,86 @@ Example response:
         }
     ]
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+
+==== Assign or unassign users from alerts
+
+Allows you to assign and unassign users from alerts. 
+
+===== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/signals/assignees`
+
+===== Request body
+
+A JSON object with either a `assignees` and `ids` field:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`assignees` |Object[] a|An array of unique identifiers (UIDs) for user profiles. Properties of the assignees object:
+
+* add: (Required, string[]) An array of assignees you want to add.
+* remove: (Required, string[]) An array of assignees you want to unassign. 
```

Comment:
Nit, just to make the phrasing consistent with the note.
```suggestion
* `remove`: (Required, string[]) An array of assignees you want to remove. 
```

Reply:
I'd prefer to keep unassign here since we use this language in the UI docs.